andes/models/group.py

```diff
@@ -438,6 +438,32 @@ def get_next_idx(self, idx=None, model_name=None):
 
         return idx
 
+    def get_idx(self):
+        """
+        Return all the devices idx in this group.
+
+        Returns
+        -------
+        list
+            List of indices.
+
+        Notes
+        -----
+        The default models sequence depends on the order of the models in the group,
+        which comes from OrderedDict `file_classes` in `models.__init__.py`.
+
+        Examples
+        --------
+        >>> ss = andes.load(andes.get_case('ieee14/ieee14_pvd1.xlsx'))
+        >>> ss.DG.get_idx()
+        [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
+
+        >>> ss.StaticGen.get_idx()
+        [2, 3, 4, 5, 6, 1]
+        """
+        all_idx = [mdl.idx.v for mdl in self.models.values()]
```

Comment:
How is this different from `list(self._idx2models.keys())`?

Reply:
Maybe rename this to `get_all_idxes`? The name `get_idx` might imply it's getting the `idx` for some inputs, but this function takes no input.